message-index/index.html

```diff
@@ -5,10 +5,10 @@
 <h2>Welcome</h2>
 
 <p>
-  This site describes the various messages that can be returned by
-  GHC, the most-commonly-used Haskell implementation, including both errors and warnings. Beginning with version
-  9.6.1, GHC emits a code with the format <code>[GHC-12345]</code> for each
-  message. These codes can be looked up below for further information.
+  This site describes the various messages that can be returned by Haskell-related tools, including both errors and warnings.
+  For example GHC, the most-commonly-used Haskell implementation, started emiting a code with the format <code>[GHC-12345]</code> for some of its
```

Comment:
```suggestion
  For example GHC, the most-commonly-used Haskell implementation, started emiting a code with the format <code>[GHC-12345]</code> for its
```
I think the "some" is needlessly pessimistic - the messages that most people encounter on a regular basis are covered, after all. I agree that "for each" is not technically correct, but I fear that the technically correct version is an overcorrection, and that strategic vagueness better communicates the state of the world.